log_extractor/log_extractor.py

```diff
@@ -457,14 +458,22 @@ def parse_logs(self):
 
 
 @click.command()
-@click.option("--job", help="Job name", required=True)
 @click.option(
-    "--build", help="build number of the job", required=True, type=int
+    "--job", cls=helper.MutuallyExclusiveOption,
+    mutually_exclusive=["skip_download", "local_log_file"],
+    help="Job name"
+)
+@click.option(
+    "--build", cls=helper.MutuallyExclusiveOption,
+    mutually_exclusive=["skip_download", "local_log_file"],
+    help="build number of the job", type=int
 )
 @click.option(
     "--folder",
-    help="Folder path to save the logs",
-    default=os.path.join(user.home, "art-tests-logs")
```

Comment:
if you will remove the `default` value, it will leave `folder` variables as `None`

Reply:
I just didn't like that we are overwriting the default value later in code, but I 'll revert this change, np.